other/install_2-1_postgresql.sql

```diff
@@ -1227,7 +1227,7 @@ CREATE INDEX {$db_prefix}messages_show_posts ON {$db_prefix}messages (id_member,
 CREATE INDEX {$db_prefix}messages_id_member_msg ON {$db_prefix}messages (id_member, approved, id_msg);
 CREATE INDEX {$db_prefix}messages_current_topic ON {$db_prefix}messages (id_topic, id_msg, id_member, approved);
 CREATE INDEX {$db_prefix}messages_related_ip ON {$db_prefix}messages (id_member, poster_ip, id_msg);
-CREATE INDEX {$db_prefix}messages_likes ON {$db_prefix}messages (likes DESC);
```

Comment:
i don't like that you want to harm postgres.
it's common that index across the different version are different,
since the database and features are differently.

Reply:
"harm" wut

Reply:
albertlast is correct that in some circumstances a DESC index can be a big improvement performance-wise:
https://www.percona.com/blog/2016/10/20/mysql-8-0-descending-indexes-can-speedup-your-queries/

My understanding is that the big benefit is if you have mixed terms in your ORDER BY clause, e.g., "ORDER BY likes DESC, name".  Which sounds like that's exactly how this index would be used.  And that is exactly what I see here:
![image](https://user-images.githubusercontent.com/23568484/79026655-63c0f400-7b3e-11ea-9158-9df1a5bdbdd0.png)

Let me think about this further...

Reply:
Actually, that was members!   This one is messages:
![image](https://user-images.githubusercontent.com/23568484/79026904-198c4280-7b3f-11ea-96bb-1e3766e28bc0.png)

Reply:
An ideal solution might be to leave the DESC in there, and add that to the list of errors that the upgrader ignores.  

I wouldn't feel comfortable doing that without being able to test it.  I don't have access to MySQL8 anywyhere at the moment, not on my hosted server nor in my WAMP.

Reply:
you can download mysql 8.0 for wamp here: http://wampserver.aviatechno.net/

Reply:
I use EasyPHP.